src/app/window-handler.ts

```diff
@@ -1277,6 +1277,7 @@ export class WindowHandler {
                 default:
                     this.url = this.globalConfig.url + `?x-km-csrf-token=${csrfToken}`;
             }
+            this.execCmd(this.screenShareIndicatorFrameUtil, []);
```

Comment:
I think we need to move all the screen sharing indicator logic to a separate handler file. 

I have included this change in https://perzoinc.atlassian.net/browse/SDA-2066 point 7.

Reply:
yes, that sounds like a reasonable improvement